### PR TITLE
Fix ol-start service only starting ol-web

### PR DESCRIPTION
**Pull request by cdrini**
Regarding #580 

The bug was from not escaping `${}` characters when echoing a bash script to a file. Resolved by creating the file in `scripts/ol-start.sh` and symlinking the `ol-start` service to this file on `vagrant provision`. This will automatically *remove the old ol-start file* when you run `vagrant provision`, so if you've edited the `ol-start` file, make sure to backup your changes, then run `vagrant provision`.

### CODE REVIEW

File: scripts/ol-start.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# a local copy of openlibrary running.
+
+sudo service nginx restart
+
+cd /openlibrary/conf/init
+for name in ol-*
+do
+	echo starting ${name//.conf}
+	initctl start ${name//.conf} || initctl restart ${name//.conf}
+done
```
Comment (tfmorris) [inline scripts/ol-start.sh]: Missing a trailing newline

Comment (cdrini) [reply scripts/ol-start.sh]: DONE

Comment (cdrini) [reply scripts/ol-start.sh]: (I wonder if there's a way to automate this...)

**Comment by mekarpeles** [comment]
this lgtm, thanks @cdrini and @tfmorris

**Pull request merged**